From a88b5e4188053ff21d3cece2ccf049d4c2685508 Mon Sep 17 00:00:00 2001
From: Li Zeng <li.zeng@intel.com>
Date: Sat, 27 Oct 2012 10:00:16 +0800
Subject: [PATCH 1/2] Add a flag
 OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled

Bug: 70693
Category: Device Enablement
Domain: AOSP-frameworks
Severity: High
Origin: Internal
Upstream-candidate: Yes

Change-Id: Iafea058d32013db2f68adeec6fadb5002c4809e5
Orig-Change-Id: I38da279d822346dddcd6805df2686992a654ed36
Signed-off-by: Li Zeng <li.zeng@intel.com>
Reviewed-on: http://android.intel.com:8080/83203
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/media/openmax/OMX_IVCommon.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/media/openmax/OMX_IVCommon.h b/include/media/openmax/OMX_IVCommon.h
index 5f9e9b6..ef57c2e 100644
--- a/include/media/openmax/OMX_IVCommon.h
+++ b/include/media/openmax/OMX_IVCommon.h
@@ -169,6 +169,10 @@ typedef enum OMX_COLOR_FORMATTYPE {
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar64x32Tile2m8ka = 0x7FA30C03,
     OMX_SEC_COLOR_FormatNV12Tiled = 0x7FC00002,
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar32m = 0x7FA30C04,
+    
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar =  0x7FA00E00,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled =  0x7FA00F00,
+ 
     OMX_COLOR_FormatMax = 0x7FFFFFFF
 } OMX_COLOR_FORMATTYPE;
 
-- 
1.7.10.4

